test/integration/generated/azureParameterGrouping/src/models/parameters.ts

```diff
@@ -124,7 +124,7 @@ export const fromParam: OperationQueryParameter = {
   parameterPath: [
     "options",
     "parameterGroupingPostReservedWordsParameters",
-    "fromParam"
+    "from"
```

Comment:
In this case, we are generating the models like 
```
export interface ParameterGroupingPostReservedWordsParameters {
  /** 'from' is a reserved word. Pass in 'bob' to pass. */
  from?: string;
  /** 'accept' is a reserved word. Pass in 'yes' to pass. */
  accept?: string;
}
```
I guess this is a valid fix. But I am kind of confused at the case in datasearch.

Reply:
This is right if you wanted to access `from` of `options` you'd do `options.parameterGroupingPostReservedWordsParameters.from` whcih matches the new parameterPath